packages/forklift-console-plugin/src/modules/Plans/views/details/hooks/useToggle.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+/**
+ * `useToggle` is a hook that manages a single boolean state value.
```

Comment:
in cases the original method didn't change, you can import it directly, no need to copy the code:
https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/src/modules/Providers/hooks/useToggle.ts#L17

for example:
`import { useToggle } from 'src/modules/Providers/hooks';
`

same with other files that are copied verbatim.

in some follow up we will move the repeating code to a common directory and if it makes sense even move it to the common package